content/docs/operations/sample-app.md

```diff
@@ -10,17 +10,17 @@ Review the following diagram to understand the setup.
 
 ```mermaid
 flowchart LR
-    A[client] -->|example.com| B[kgateway proxy]
+    A[client] -->|example.com| B[{{< reuse "/docs/snippets/kgateway.md" >}} proxy]
     B --> C[httpbin backend]
 ```
 
 * The client calls the `www.example.com` hostname that you set up in the Gateway configuration.
-* The kgateway proxy receives the request. Based on the routing rules that you set up in the Gateway configuration, the kgateway proxy forwards the traffic to the backend destination, which is the httpbin service. The kgateway proxy is available from an external LoadBalancer service that is backed by an IP address that your cloud provider typically assigns. For testing in a local cluster where you do not have an external service, you can enable port-forwarding so that the kgateway proxy listens on the localhost instead.
-* The httpbin service receives and responds to the request. Note that the httpbin service does not have to be publicly exposed because the kgateway proxy handles the external traffic. Instead, it can have an internal service type, such as ClusterIP.
+* The {{< reuse "/docs/snippets/kgateway.md" >}} proxy receives the request. Based on the routing rules that you set up in the Gateway configuration, the {{< reuse "/docs/snippets/kgateway.md" >}} proxy forwards the traffic to the backend destination, which is the httpbin service. The {{< reuse "/docs/snippets/kgateway.md" >}} proxy is available from an external LoadBalancer service that is backed by an IP address that your cloud provider typically assigns. For testing in a local cluster where you do not have an external service, you can enable port-forwarding so that the {{< reuse "/docs/snippets/kgateway.md" >}} proxy listens on the localhost instead.
```

Comment:
I always changed it to gateway proxy to avoid a conref. but we can also leave it as-is